src/web/core/src/main/resources/GeoServerApplication_ca.properties

```diff
@@ -21,23 +21,23 @@ AbstractDataAccessPage.cantGetDataStoreFactory = No s'ha trobat la factoria del
 AbstractDataAccessPage.dataSrcName             = Nom de l'origen de dades
 AbstractDataAccessPage.workspace               = Espai de treball
 
-AbstractLayerGroupPage.bounds                  = L\u00edmits
-AbstractLayerGroupPage.chooseLayer             = Sel\u00b7lecciona una nova capa
-AbstractLayerGroupPage.chooseStyle             = Sel\u00b7lecciona un estil alternatiu
-AbstractLayerGroupPage.description             = Edita els continguts d'un grup de capes
-AbstractLayerGroupPage.down                    = Mou la capa a sota
-AbstractLayerGroupPage.duplicateGroupNameError = Ja existeix un grup de capes anomenat ${name}
-AbstractLayerGroupPage.notFound                = No s'ha trobat el grup de capes "{0}"
-AbstractLayerGroupPage.oneLayerMinimum         = El grup de capes ha de contenir al menys una capa
-AbstractLayerGroupPage.th.code                 = Codi\u0009
-AbstractLayerGroupPage.th.defaultStyle         = Estil per defecte
-AbstractLayerGroupPage.th.description          = Descripci\u00f3
-AbstractLayerGroupPage.th.layer                = Capa
-AbstractLayerGroupPage.th.position             = Posici\u00f3
-AbstractLayerGroupPage.th.remove               = Esborra
-AbstractLayerGroupPage.th.style                = Estil
-AbstractLayerGroupPage.title                   = Grup de capes
-AbstractLayerGroupPage.up                      = Mou la capa a sobre
+LayerGroupEditPage.bounds                  = L\u00edmits
```

Comment:
Hmmm okay good point. This could be one reason to leave the class names intact. Let me know.